pkg/pushsync/pushsync.go

```diff
@@ -130,6 +143,54 @@ func (ps *PushSync) handler(ctx context.Context, p p2p.Peer, stream p2p.Stream)
 				return fmt.Errorf("chunk store: %w", err)
 			}
 
+			// push the chunk to the closest peers in parallel for replication
+			// any errors here should NOT impact the rest of the handler
+			var skipPeers []swarm.Address
+			for i := 0; i < nPeersToPushsync; i++ {
+				peer, err := ps.closestPeer(chunk.Address(), skipPeers, true)
+				if err != nil {
+					ps.logger.Tracef("closest peer: %w", err)
```

Comment:
consider adding a bit more context here so that we know where it came from. See `pushsync.go#L256`. The same holds for all the other loglines that were added